### PR TITLE
put workflows in proper fs directory

### PR DESCRIPTION
**Pull request by btylerburton**

### CODE REVIEW

File: Rakefile
Comment (btylerburton) [reply Rakefile]: it's a ruby task runner.

Comment (nickumia-reisys) [reply Rakefile]: I guess what's the benefit we're trying to add by using it?

File: Rakefile
Comment (nickumia-reisys) [inline Rakefile:19]: Is necessary because of the federalist bug of links being broken?  Or is this for something else?

**Comment by btylerburton** [comment]
ready to merge once this is merged: https://github.com/GSA/data.gov/pull/4123

File: README.md
Comment (nickumia-reisys) [inline README.md:27]: Is the alias important?  I feel like it's too much detail for the README

Comment (btylerburton) [reply README.md:27]: i'm fine with taking them out.

**Comment by btylerburton** [comment]
Looks like remote workflows need to specify a branch @nickumia-reisys 
- https://github.com/GSA/us-data-federation/actions/runs/3698810027/workflow 
- https://stackoverflow.com/questions/73026597/github-actions-suddenly-throwing-no-version-specified-for-reusable-workflows-t

**Review by nickumia-reisys — APPROVED** [review]
Makes sense ☁️

**Pull request merged**
